src/System.Private.Xml/src/System/Xml/Core/XmlReaderSettings.cs

```diff
@@ -569,7 +569,7 @@ private void Initialize(XmlResolver resolver)
 
         private static XmlResolver CreateDefaultResolver()
         {
-            return new XmlSystemPathResolver();
+            return new XmlUrlResolver();
```

Comment:
did you try to run the tests against UAP? nothing to worry more than just double checking